servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H1ProtocolConfig.java

```diff
@@ -87,4 +87,13 @@ default String alpnId() {
      * <a href="https://tools.ietf.org/html/rfc7230#section-4.1.2trailers">trailer fields</a>
      */
     int trailersEncodedSizeEstimate();
+
+    /**
+     * Additional extensions for <a href="https://tools.ietf.org/html/rfc7230">HTTP/1.1</a> specification that help to
+     * relax constrains for backward compatibility with older systems.
```

Comment:
I would suggest to remove the words after `specification` above. "exceptions for specifications" is sufficient.